packages/@webex/internal-plugin-metrics/test/unit/spec/new-metrics.ts

```diff
@@ -83,6 +83,24 @@ describe('internal-plugin-metrics', () => {
       webex.internal.newMetrics.isReadyToSubmitBusinessEvents();
       assert.isDefined(webex.internal.newMetrics.businessMetrics);
     })
+
+    it('passes the table through to the business metrics', () => {
+      assert.isUndefined(webex.internal.newMetrics.businessMetrics)
+      webex.internal.newMetrics.isReadyToSubmitBusinessEvents();
+      assert.isDefined(webex.internal.newMetrics.businessMetrics);
+      webex.internal.newMetrics.businessMetrics.submitBusinessEvent = sinon.stub();
+      webex.internal.newMetrics.submitBusinessEvent({
+        name: 'foobar',
+        payload: {},
+        table: 'test',
```

Comment:
Is it a problem that this isn't one of the ENUMs listed in the types for Table?

Reply:
no it does not for 2 reasons

1. this is the interface, the interface has no logic as it pertains to passing the value through, those tests are underneath
2. the tests (I don't like this) test against the dist javascript, where types are stripped, so testing that ts doesn't do anything